packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -167,17 +167,22 @@ const OpenQueueCard = ({
         </HeaderDiv>
         <RightQueueInfoRow>
           <Space direction="vertical" align="end" size="middle">
-            {!queue.allowQuestions && (
-              <Tooltip title="This queue is no longer accepting questions">
-                <Tag
-                  icon={<StopOutlined />}
-                  color="error"
-                  style={{ margin: 0 }}
-                >
-                  Not Accepting Questions
-                </Tag>
-              </Tooltip>
-            )}
+            <QueueInfoTags>
+              {queue?.isProfessorQueue && (
+                <Tag color="blue">Professor Queue</Tag>
```

Comment:
i think you'd want to include a `style={{ margin: 0 }}` here because there's a small indent here
![image](https://user-images.githubusercontent.com/23366632/165010136-4299371d-abfb-4c6f-a0a1-8bb8aa722ded.png)

Reply:
might be worth abstracting that 🤔, if possible ofc

Reply:
Good catch, fixed. By abstracting, do you mean like making that type of tag a styled component or something else

Reply:
Oh ye smth like that, but i think that change would be a nit. So not a big deal imo